Compile error, after release of Dual Setpoint

Have a working esp8266 connected to a Mitsubishi MSY-GE15NA-8
Attempting to update today, and run into this error:  

```
Processing ac-diningrm (board: d1_mini; framework: arduino; platform: platformio/espressif8266@4.2.1)
--------------------------------------------------------------------------------
HARDWARE: ESP8266 80MHz, 80KB RAM, 4MB Flash
Dependency Graph
|-- ESPAsyncTCP @ 2.0.0
|-- ESP8266WiFi @ 1.0
|-- ESPAsyncWebServer @ 3.7.10
|-- DNSServer @ 1.1.1
|-- ESP8266mDNS @ 1.2
|-- noise-c @ 0.1.10
Compiling .pioenvs/ac-diningrm/src/esphome/components/cn105/climateControls.cpp.o
Compiling .pioenvs/ac-diningrm/src/esphome/components/cn105/heatpumpFunctions.cpp.o
src/esphome/components/cn105/climateControls.cpp: In member function 'void esphome::CN105Climate::controlDelegate(const esphome::climate::ClimateCall&)':
src/esphome/components/cn105/climateControls.cpp:149:60: error: expected primary-expression before ',' token
  149 |         ESP_LOGD("control", "sanitizing dual setpoints..."),
      |                                                            ^
*** [.pioenvs/ac-diningrm/src/esphome/components/cn105/climateControls.cpp.o] Error 1
========================== [FAILED] Took 3.44 seconds ==========================
```


This is even before I try adding the "supports:" section to the Climate config.
Complete config:

```
substitutions:
  name: ac-diningrm
  friendly_name: Dining Room AC
  remote_temp_sensor: climate.living_room

esphome:
  name: ${name}
  friendly_name: ${friendly_name}

# For ESP8266 Devices
esp8266:
  board: d1_mini

uart:
  id: HP_UART
  baud_rate: 2400
  tx_pin: 1
  rx_pin: 3

# Enable Home Assistant API
api:
  encryption:
    key: "redacted"

ota:
  password: !secret ota_password
  platform: esphome

wifi:
  ssid: !secret wifi_ssid
  password: !secret wifi_password

  # Enable fallback hotspot (captive portal) in case wifi connection fails
  ap:
    ssid: "Ac-Diningrm Fallback Hotspot"
    password: "372MmiMV5AcW"

external_components:
  - source: github://echavet/MitsubishiCN105ESPHome
    # refresh: 0s

captive_portal:
    
  # Sync time with Home Assistant.
time:
  - platform: homeassistant
    id: homeassistant_time

# Text sensors with general information.
text_sensor:
  # Expose WiFi information as sensors.
  - platform: wifi_info
    ip_address:
      name: IP

# Sensors with general information, and remote sensor.
sensor:
  # Uptime sensor.
  - platform: uptime
    name: Uptime
  #
  - platform: homeassistant
    name: "Remote Temperature Sensor"
    entity_id: ${remote_temp_sensor} # Replace with your HomeAssistant remote sensor entity id, or include in substitutions
    attribute: current_temperature
    internal: false
    disabled_by_default: true
    device_class: temperature
    state_class: measurement
    unit_of_measurement: "°C"
    filters:
    # Uncomment the lambda line to convert F to C on incoming temperature
      - lambda: return (x - 32) * (5.0/9.0);
      - clamp: # Limits values to range accepted by Mitsubishi units
          min_value: 16
          max_value: 31
          ignore_out_of_range: true
      - throttle: 30s
    on_value:
      then:
        - logger.log:
            level: INFO
            format: "Remote temperature received from HA: %.1f C"
            args: [ 'x' ]
        - lambda: 'id(hp).set_remote_temperature(x);'

# Enable logging
logger:
  hardware_uart: UART1 # Only needed on ESP8266 devices
  level: INFO
  logs:
    EVT_SETS : INFO
    WIFI : INFO
    MQTT : INFO
    WRITE_SETTINGS : INFO
    SETTINGS : INFO
    STATUS : INFO
    CN105Climate: WARN
    CN105: INFO
    climate: WARN
    sensor: WARN
    chkSum : INFO
    WRITE : WARN
    READ : WARN
    Header: INFO
    Decoder : INFO
    CONTROL_WANTED_SETTINGS: INFO
#  level: DEBUG
#  logs:
#    EVT_SETS : DEBUG
#    WIFI : INFO
#    MQTT : INFO
#    WRITE_SETTINGS : DEBUG
#    SETTINGS : DEBUG
#    STATUS : INFO
#    CN105Climate: WARN
#    CN105: DEBUG
#    climate: WARN
#    sensor: WARN
#    chkSum : INFO
#    WRITE : WARN
#    READ : WARN
#    Header: INFO
#    Decoder : DEBUG
#    CONTROL_WANTED_SETTINGS: DEBUG


# Create a button to restart the unit from HomeAssistant. Rarely needed, but can be handy.
button:
  - platform: restart
    name: "Restart ${friendly_name}"

climate:
  - platform: cn105
    id: hp
    name: "${friendly_name}"
    icon: mdi:heat-pump
    # supports:
    #   mode: [COOL, FAN_ONLY, DRY, AUTO]
    visual:
      min_temperature: 16
      max_temperature: 31
      temperature_step:
        target_temperature: 1
        current_temperature: 0.1
    # Fahrenheit compatibility mode - uses Mitsubishi's "custom" unit conversions, set to
    # "true" for better support of Fahrenheit units in HomeAssistant
    fahrenheit_compatibility: false
    # Timeout and communication settings
    remote_temperature_timeout: 20min
    update_interval: 3s
    debounce_delay : 500ms
    compressor_frequency_sensor:
      name: Compressor Frequency
    vertical_vane_select:
      name: Vertical Vane
    stage_sensor:
      name: Stage
      entity_category: diagnostic
    sub_mode_sensor:
      name: Sub Mode
      entity_category: diagnostic
    auto_sub_mode_sensor:
      name: Auto Sub Mode
      entity_category: diagnostic
    # input_power_sensor:
    #   name: Input Power
    # kwh_sensor:
    #   name: Energy Usage
    # runtime_hours_sensor:
    #   name: Runtime Hours
    #   entity_category: diagnostic
    #   disabled_by_default: true


`

--- a/components/cn105/climateControls.cpp
+++ b/components/cn105/climateControls.cpp
@@ -146,8 +146,8 @@ void CN105Climate::controlDelegate(const esphome::climate::ClimateCall& call) {
             }
         }
         updated = true;
-        ESP_LOGD("control", "sanitizing dual setpoints..."),
-            this->sanitizeDualSetpoints();
+        ESP_LOGD("control", "sanitizing dual setpoints...");
+        this->sanitizeDualSetpoints();
         ESP_LOGD("control", "sanitized dual setpoints: low: %.1f - high: %.1f", this->target_temperature_low, this->target_temperature_high);
         this->controlTemperature();
         ESP_LOGD("control", "controlled temperature to: %.1f", this->wantedSettings.temperature);

